code/modules/mob/living/carbon/human/life.dm

```diff
@@ -340,8 +343,49 @@
 	// Tissues die without blood circulation
 	adjustBruteLoss(2)
 
-
-
+/mob/living/carbon/human/proc/handle_bones()
+	for(var/obj/item/bodypart/OB in bodyparts)
+		var/currentzone = OB.body_zone
+		var/list/currentbones = list()
+		for(var/obj/item/organ/bone/B in getorganszone(currentzone))
+			currentbones += B
+		if(!currentbones.len)
+			if(prob(66))
+				for(var/obj/item/organ/O in getorganszone(currentzone))
+					src.adjustOrganLoss(O, rand(3,10))
+				src.apply_damage(2, BRUTE, currentzone, FALSE, TRUE)
+				if(prob(25))
+					to_chat(src, "<span class='userdanger'>Your [get_bodypart(currentzone)] is squishy and hurts like fucking hell! What the fuck?!</span>")
```

Comment:
You don't need to have so many expletives; _"Your [get_bodypart(currentzone)] is squishy and hurts like hell!"_works just fine.